src/api/viewportFeatures.js

```diff
@@ -50,18 +50,18 @@ function addIntersectedFeaturesInTile({ map, tile, viewport, uniqueIdProperty })
 }
 
 function isTileFullVisible(bbox, viewport) {
-  const [minX, maxX, maxY, minY] = viewport;
+  const [minX, minY, maxX, maxY] = viewport;
```

Comment:
was this a bug?... if we had one, we could probably change it for an object

Reply:
This is how we are saving the viewport in the store. It's ok if you want to change it by an object but it's a big refactor and it should be done in other PR